draft-ietf-masque-h3-datagram.md

```diff
@@ -241,7 +241,9 @@ negotiate the same draft version.
 ## HTTP Datagrams using Capsules
 
 When HTTP/3 Datagrams are unavailable or undesirable, HTTP Datagrams can be sent
-using the Capsule Protocol, see {{datagram-capsule}}.
+using the Capsule Protocol, see {{datagram-capsule}}. Capsules can also be used
```

Comment:
Adding a note to explain how capsules will be used in things like CONNECT-IP, may partly address #159.

Reply:
I like this note, but it doesn't make sense in the "HTTP Datagrams using Capsules" section. Can we move it the Capsule Protocol section?

Reply:
Moved down